docs/versioning.md

```diff
@@ -0,0 +1,48 @@
+<!--
+   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
+   -
+   - SPDX-License-Identifier: LicenseRef-MIT-TQ
+   -->
+
+# Versioning
+
+`tezos-packaging` provides support for all Octez stable releases and RC candidates.
+
+However, in some cases we provide multiple releases within the same upstream version
+because our packages provide additional functionality (e.g. systemd services in Ubuntu
+and Fedora packages or brew formulae with launchd services) that may need to be updated
+within the single upstream release.
```

Comment:
Minor suggestion, buy I would rephrase this to more have this order instead:
1. tezos-packaging releases can provide additional functionalities
2. give examples
3. (implied) these are tracked in addition top of the upstream version


P.s it sounds more correct to me to say "the same" rather than "the single" at the end there